cinder/volume/drivers/vmware/vmdk.py

```diff
@@ -782,6 +787,15 @@ def _initialize_connection(self, volume, connector):
         :return: Return connection information
         """
         backing = self.volumeops.get_backing(volume.name, volume.id)
+
+        # Check that connection_capabilities match
+        # This ensures the connector is bound to the same v-center service
+        if 'connection_capabilities' in connector:
+            for capability in self._get_connection_capabilities():
+                if capability not in connector['connection_capabilities']:
+                    raise exception.ConnectorRejected(
+                        reason="%s not found in the connector." % capability)
```

Comment:
Why don't you use `set()` here, like above already? I like sets. If it's because of the Exception-message, you can do one more set operation to find out which capabilities are missing.

Reply:
I think it's because... this was one of the first blocks written in this commit, when maybe I was not 100% sure how to compare the capabilities, thus I started with a `for` and stayed like that.